### PR TITLE
Trio

### PR DESCRIPTION
**Pull request by davidalencar**
## New Company Prerequisites

- [ ] Industry must be one of the 11 sectors of the [GICS](https://www.msci.com/our-solutions/indexes/gics).
- [ ] The description makes some mention of Elixir usage.

### CODE REVIEW

**Comment by jsegal205** [comment]
Hi @davidalencar Thanks for the contribution 🎉 ! 

Can you check and remedy the CI failures and then we can get this merged in?

File: priv/companies/trio.exs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      city: "Curitiba",
+      state: "Paraná",
+      country: "Brazil"
+    },
+    description: """
+    Trio is a financial institution specializing in instant payments in Brazil. All our products, from the banking core to the payment checkout, were developed using Elixir, Phoenix, and LiveView.
+    """,
+    last_changed_on: ~D[2024-11-15]
+  }
+
```
Comment (jsegal205) [inline priv/companies/trio.exs:17]:
```suggestion
%{
  name: "Trio",
  website: "https://www.triohq.com/en",
  github: "https://github.com/trioHQ",
  # reference lib/companies/industries.ex for a list of recommended industries to use here
  industry: "Financials",
  location: %{
    city: "Curitiba",
    state: "Paraná",
    country: "Brazil"
  },
  description: """
  Trio is a financial institution specializing in instant payments in Brazil. All our products, from the banking core to the payment checkout, were developed using Elixir, Phoenix, and LiveView.
  """,
  last_changed_on: ~D[2024-11-15]
}
```

Comment (jsegal205) [inline priv/companies/trio.exs:17]:
```suggestion
```suggestion
%{
  name: "Trio",
  website: "https://www.triohq.com/en",
  github: "https://github.com/trioHQ",
  # reference lib/companies/industries.ex for a list of recommended industries to use here
  industry: "Financials",
  location: %{
    city: "Curitiba",
    state: "Paraná",
    country: "Brazil"
  },
  description: """
  Trio is a financial institution specializing in instant payments in Brazil. All our products, from the banking core to the payment checkout, were developed using Elixir, Phoenix, and LiveView.
  """,
  last_changed_on: ~D[2024-11-15]
}
```
```

Comment (doomspork) [reply priv/companies/trio.exs]: Thank you @jsegal205

**Review by jsegal205 — APPROVED** [review]

**Pull request merged**